packages/fxa-shared/sentry/report-validation-error.ts

```diff
@@ -0,0 +1,56 @@
+import * as Sentry from '@sentry/node';
```

Comment:
By breaking this out into its own file separate from index.ts, the context issue was resolved. We must be careful not to accidentally include this file in ./index.ts. Doing so could result in @sentry/node being required on the client side which completely breaks things. Additionally, once the file was separated the strange issue I had with mocks on sentry also cleared up.